wiki/Skinning/Interface/en.md

```diff
@@ -532,8 +532,8 @@ The following table is for all game modifier icons.
 **Notes:**
 
 - This mod is used for [osu!standard](/wiki/osu!standard), [osu!taiko](/wiki/osu!taiko), and [osu!catch](/wiki/osu!catch).
-- This mod is called _Relax_.
-  - This mod will auto-click for the player, while the player just needs to move the cursor.
+- This mod has different effects when using it with different modes:
+  - In [osu!standard](/wiki/osu!standard), this mod will auto-click for the player, while the player just needs to move the cursor.
```

Comment:
It automatically clicks for you so you can rest your clicking fingers :/ (but you still have to move the cursor)
Trivia: you can still tap in relax, it just doesn't do much since the auto-clicking taps for you.